### ISSUE TITLE
Plan Dump from Latex

### ISSUE THREAD

**Issue by mbaddar1** [mbaddar1/genmodel#6]
\subsection{Work-Schedule}\label{subsec:time-table}
    \begin{enumerate}
        \item Read about score/gradient matching based generative models~\cite{Generati61:online} 20-Nov-2023 \textcolor{cyan}{DONE}
        \item Read this comprehensive paper :\cite{song2021scorebased} 20 to 25 Nov 2023 \textcolor{green}{In progress}
        \begin{enumerate}
            \item Focus on section 2.1 (score-matching) \textcolor{cyan}{done}
            \item Focus on section 2.2 (DDPM) \textcolor{green}{in prgoress}
            \item Skim through the rest
        \end{enumerate}
        \item A quick overview for this diffusion model tutorial~\cite{weng2021diffusion} 22-Nov-2023 -> Focus on
        Loss function understanding and derivation , also on different variants of it
        \item Experiment these two simple implementation of DDPM~\cite{aju22DDP15:online,Generati57:online}.
            Have a look at this bigger one~\cite{lucidrai97:online}
        \item Hands on coding for the basic score matching~\cite{DBLP:journals/corr/abs-1907-05600}
        From scratch implementation can be good for learning~\cite{Benefits95:online}. \newline
        Try this tutorial \cite{Tutorial71:online} which I have found in this GitHub page~\cite{yangsong73:online}
        and blog post \cite{Generati61:online}
        =============================
        \item A quick overview for Deep Learning Book~\cite{jentzen2023mathematical}
        \textbf{Focus on Chapter 5 -> Gradient-FLow} 25-Nov-2023
        \item A quick overview Riemannian Optimization~\cite{smith2014optimization}
        \item Check this paper for TT-DNN \cite{qi2022exploiting} (Good One)
        \item Check this TT-DNN paper~\cite{qi2019riemannian} (Rejected one)
        \item TT-GP \cite{izmailov2018scalable}
        \item A quick overview for this Riemannian Optimization Autograd paper~\cite{novikov2021automatic} 26-Nov-2023
        \item Riemannian Optimization with T3F lib \cite{novikov2020tensor}
        \item A quick overview for this Diffusion Model survey paper~\cite{yang2023diffusion} 27-Nov-2023
        \item Try to make connections between topics from Riemannian Opt or DL Book to Diffusion Models 27 to 28 Nov 2023
        \item Experimentation the score sde method(\href{https://github.com/yang-song/score_sde}{code})
        and DDPM (\href{https://github.com/lucidrains/denoising-diffusion-pytorch}{code}).
        29 Nov 2023
    \end{enumerate}
    %%%
    \subsubsection{Understanding Connection Between Diffusion Models, Langevin Dynamic and Differential Equations}
    \label{subsubsec:conn-ddpm-lavdyn-diffeq}
    \begin{itemize}
        \item write the sampling eqns of both langevin dyn , ddpm \cite{weng2021diffusion}
        \item compare {$\epsilon_{\theta}$} and  $\nabla_{x} \log p(x)$ .
        \item Highlight that the derivation of correspondence between DDPM and SMLD losses is heuristic based
        Eq (3) in \cite{song2021scorebased} and Eq(12) in \cite{ho2020_ddpm}
    \end{itemize}
